server/api/api.go

```diff
@@ -133,7 +125,7 @@ func (a *API) panicHandler(next http.Handler) http.Handler {
 					mlog.String("stack", string(debug.Stack())),
 					mlog.String("uri", r.URL.Path),
 				)
-				a.errorResponse(w, r.URL.Path, http.StatusInternalServerError, "", nil)
+				a.customErrorResponse(w, r.URL.Path, http.StatusInternalServerError, "", nil)
```

Comment:
makes sense to use a `errorResponse(w, r, errors.New('panic during request handling'))` or something like that?